src/components/virtualRepeat/virtual-repeater.js

```diff
@@ -76,7 +78,7 @@ var MAX_ELEMENT_SIZE = 1533917;
 var NUM_EXTRA = 3;
 
 /** @ngInject */
-function VirtualRepeatContainerController($$rAF, $scope, $element, $attrs) {
+function VirtualRepeatContainerController($$rAF, $scope, $element, $attrs, $parse) {
```

Comment:
Nitpick: please put $parse before $scope in the params list (I prefer it to go services then locals)